### PR TITLE
INT-133: Fix issue by using mb_pathinfo instead of pathinfo

### PR DESCRIPTION
**Pull request by inez**
PHP pathinfo does not work wiht multibyte strings - it's a known issue that is solved in PHP6. In the meantime I decided to use mb_pathinfo which I found here http://php.net/manual/en/function.pathinfo.php#107461 and for which implementation looks pretty solid.

### CODE REVIEW

**Comment by inez** [comment]
@Grunny @mari-mcmurtrie @owend

**Comment by inez** [comment]
Before you say: I know, it would be awesome to have units for it.

File: includes/wikia/GlobalFunctions.php [hunk 1]
```diff
@@ -1613,3 +1613,15 @@ function wfGetUniqueArrayCI( array $arr ) {
 	$unique = array_intersect_key( $arr, array_unique( $lower ) );
 	return array_filter( $unique );
 }
+
+/**
+ * Like pathinfo but with support for multibyte - copied from http://php.net/manual/en/function.pathinfo.php#107461
+ */
+function mb_pathinfo($filepath) {
```
Comment (Grunny) [inline includes/wikia/GlobalFunctions.php]: I know this is copied from elsewhere, but I still think it should use our coding conventions (spacey, always use braces on blocks, etc.).

**Comment by Grunny** [comment]
Other than the coding convention comment, LGTM. So, can be merged once unit tested and any extra manually testing is done.

**Comment by inez** [comment]
@Grunny Ok. This should be ready for review again.

**Comment by Grunny** [comment]
LGTM.

**Pull request merged**